lib/modules/platform/bitbucket-server/readme.md

```diff
@@ -5,9 +5,11 @@
 First, create a [HTTP access token](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html) for the bot account.
 Let Renovate use your HTTP access token by doing _one_ of the following:
 
-- Set your HTTP access token as a `password` in your `config.js` file
-- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
-- Set your HTTP access token when you run Renovate in the CLI with `--password=`
+- Set your HTTP access token as a `token` or `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_TOKEN` or `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--token=` or `--password=`
```

Comment:
ah, sorry. the token should be set as token (preferred). so we can drop the info to also use password here.

can we detect if a access token is set as password and log a deprecation warning? 🤔

it seems you can't use a password since some versions?

Reply:
Why should we log a deprecation warning? Both ways are valide to be used in BB and its not deprecated on their side.

Reply:
sure, but access token should be passed ax token instead of password inside renovate, so that's why I suggest to drop at least the docx part to use them as password. even we still support that

Reply:
☝️